test/validation/api/atomic/atomic.c

```diff
@@ -18,15 +18,18 @@
 #define ADD_SUB_CNT		5
 
```

Comment:
Copyright year could be updated.

Also, old code, but `VERBOSE` and `MAX_ITERATIONS` seem to be unused and could be removed.